apps/web/pages/settings/organizations/[id]/onboard-admins.tsx

```diff
@@ -0,0 +1,37 @@
+import { useRouter } from "next/router";
+
+import { AddNewOrgAdminsForm } from "@calcom/features/ee/organizations/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { WizardLayout, Meta } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+const OnboardTeamMembersPage = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  if (!router.isReady) return null;
+  return (
+    <>
+      <Meta
+        title={t("invite_organization_admins")}
+        description={t("invite_organization_admins_description")}
+      />
+      <AddNewOrgAdminsForm />
+    </>
+  );
+};
+
+OnboardTeamMembersPage.getLayout = (page: React.ReactElement) => (
+  <WizardLayout
+    currentStep={4}
+    maxSteps={5}
+    isOptionalCallback={() => {
+      window.location.replace(`/getting-started`);
+    }}>
```

Comment:
same as above why can't we use router here

Reply:
Didn’t know it was available from getLayout, done!